extension/src/popups/MinervaMenuRenderInfo.js

```diff
@@ -0,0 +1,25 @@
+function renderStudentRankings(classRankingBody, response) {
+    response.sort((a, b) => {
+        return parseInt(a.time) < parseInt(b.time) ? 1: -1;
```

Comment:
This looks like a mistake on my part. I patched the server to return integers instead #84 

Also, If you are comparing integers already, are `1: -1` part needed?

Reply:
This was when the server was returning strings.